R/calc_aggregate_counts.R

```diff
@@ -29,6 +29,49 @@
 #' calc_aggregate_counts(state = TRUE, all_dates = TRUE)
 #' @export
 
+create_statewide_counts <- function(dat, all_dates) {
```

Comment:
I have two nitpicks here: 
1. I'd like this new function to come after calc_aggregate_counts. My logic here is that you should be able to read through the main function first, then dig into supplementary functions as needed/if you care. 
2. I'd like to use the [naming convention that Google uses](https://google.github.io/styleguide/Rguide.html) for a private/helper function, which is starting the name of the function with a `.`. I like the way this delineates a core function from a helper function, but if you have strong feelings, let's chat!